steamapi/user.py

```diff
@@ -205,7 +205,7 @@ def country_code(self):
         """
         :rtype: str
         """
-        return self._summary.loccountrycode
+        return getattr(self._summary, 'loccountrycode', "Unknown")
```

Comment:
Return `None` when no valid answer is available. (And change `rtype` to `str or NoneType`)